### PR TITLE
Removed commented code

### PR DESCRIPTION
**Pull request by fballiano**
In our source code there are many commented debug lines, IMHO they should be removed.

### CODE REVIEW

**Review by justinbeaty — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

File: app/design/frontend/rwd/default/template/downloadable/catalog/product/type.phtml [hunk 1]
```diff
@@ -39,5 +39,3 @@
     </p>
 <?php endif ?>
 <?php echo $this->getChildHtml('product_type_data_extra') ?>
-<?php //echo $this->getPriceHtml($_product) ?>
```
Comment (kiatng) [inline app/design/frontend/rwd/default/template/downloadable/catalog/product/type.phtml:42]: See the base template: https://github.com/OpenMage/magento-lts/blob/ee46ff9b9a45809d5ebc7b09c13a91844f7c402c/app/design/frontend/base/default/template/downloadable/catalog/product/type.phtml#L36-L37

I would keep it for reference.

File: app/design/adminhtml/default/default/template/catalog/category/tree.phtml [hunk 2]
```diff
@@ -30,8 +30,6 @@
     <?php echo $this->getStoreSwitcherHtml();?>
     <div class="tree-actions">
         <?php if($this->getRoot()): ?>
-            <?php //echo $this->getCollapseButtonHtml() ?>
-            <?php //echo $this->getExpandButtonHtml() ?>
```
Comment (kiatng) [inline app/design/adminhtml/default/default/template/catalog/category/tree.phtml:34]: There are references in https://github.com/OpenMage/magento-lts/blob/76d6e6a0391a13269e3137bedbcf0c8b30472d9a/app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tree.php#L117-L125

which are not used anywhere. Should that be deleted too?

File: app/design/frontend/rwd/default/template/downloadable/catalog/product/type.phtml (hunk 1 above)
Comment (fballiano) [reply app/design/frontend/rwd/default/template/downloadable/catalog/product/type.phtml:42]: restored

File: app/design/adminhtml/default/default/template/catalog/category/tree.phtml (hunk 2 above)
Comment (fballiano) [reply app/design/adminhtml/default/default/template/catalog/category/tree.phtml:34]: right, I checked and remove those 2 methods

File: app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tree.php [hunk 3]
```diff
@@ -114,16 +114,6 @@ public function getAddSubButtonHtml()
         return $this->getChildHtml('add_sub_button');
     }
 
-    public function getExpandButtonHtml()
-    {
-        return $this->getChildHtml('expand_button');
```
[the remark this answers is not in the document]
Comment (fballiano) [reply app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tree.php]: I thought they were pretty safe to remove cause it's an adminhtml block, anyway i've restored them

**Review by kiatng — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Comment by fballiano** [comment]
merged and v20ed, thanks!

**Pull request merged**